src/utils/api/apiFetch.ts

```diff
@@ -9,6 +9,7 @@ import { ProgramResponse } from "./ministrySchool/interfaces";
 import { DepartmentType } from "./settings/departmentInterfaces";
 import { PositionType } from "./settings/positionInterfaces";
 import { VisitorDetailsType, VisitorType } from "./visitors/interfaces";
+import { LifeCenterType } from "@/pages/HomePage/pages/LifeCenter/LifeCenter";
```

Comment:
no 
see where you are importing it from 
this component shouldn' t even know there is something call homepage